src/components/Checklist.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from "react";
+
+import Checkbox from "@codegouvfr/react-dsfr/Checkbox";
+
+import { Domaine } from "@/models/member";
+
+export function markdownLinksToReact(markdown: string): React.ReactNode[] {
+    const regex = /\[([^\]]+)\]\((https?:\/\/[^\)]+)\)/g;
+    const parts: React.ReactNode[] = [];
+
+    let lastIndex = 0;
+    let match;
+
+    while ((match = regex.exec(markdown)) !== null) {
+        const [fullMatch, text, url] = match;
+        const index = match.index;
+
+        // Push any text before the link
+        if (lastIndex < index) {
+            parts.push(markdown.slice(lastIndex, index));
+        }
+
+        // Push the actual <a> element
+        parts.push(<> </>);
+        parts.push(
+            <>
+                <a
+                    key={index}
+                    href={url}
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    className="fr-link"
+                >
+                    {text}
+                </a>
+            </>
+        );
+        parts.push(<> </>);
+        lastIndex = index + fullMatch.length;
+    }
+
+    // Push any remaining text after the last match
+    if (lastIndex < markdown.length) {
+        parts.push(markdown.slice(lastIndex));
+    }
+
+    return parts;
+}
```

Comment:
C'est pour faire du markdown to html ? je crois qu'on a déjà des dépendances qui font ça

Reply:
on a mdParser, mais il met tout dans une balise <p>, par simplissité pour l'instant j'ai fait ça, mais faut en effet regarder dans le détail si on a pas quelque chose qui marche déjà.

Reply:
yes, j'avais pas trop cherché, mais il y a un renderInline 👍